example/todomvc/store/storeutil/storeutil.go

```diff
@@ -24,7 +22,7 @@ func (r *ListenerRegistry) Add(key interface{}, listener func()) {
 		key = new(int)
 	}
 	if _, ok := r.listeners[key]; ok {
-		panic(fmt.Sprintf("listener with key already exists: %v", key))
+		panic("duplicate listener key")
```

Comment:
should still include key here, otherwise the panic message is substantially less useful.

Reply:
key is an interface, could be anything (struct, pointer, whatever).

Reply:
Also, this code is in the examples, and will probably go away at some point anyway, which is why I didn't worry too much about it.